transport_nantes/transport_nantes/settings.py

```diff
@@ -169,44 +170,50 @@
             'class': 'logging.StreamHandler',
             'formatter': 'django.server',
         },
-        #'mail_admins': {
-        #    'level': 'ERROR',
-        #    'filters': ['require_debug_false'],
-        #    'class': 'django.utils.log.AdminEmailHandler'
-        #},
+        'mail_admins': {
+           'level': 'ERROR',
+           'filters': ['require_debug_false'],
+           'class': 'django.utils.log.AdminEmailHandler'
+        },
         'file': {
             'level': 'INFO',
-            'class': 'logging.FileHandler',
+            'class': 'logging.handlers.TimedRotatingFileHandler',
             'filename': settings_local.LOG_DIR + "tn_web.log",
             'formatter': 'django.server',
+            'when': 'd',
+            'utc': True,
+            'backupCount': 10,
         },
         'django_ses_notification': {
             'level': 'INFO',
-            'class': 'logging.FileHandler',
+            'class': 'logging.handlers.TimedRotatingFileHandler',
             'filename': settings_local.LOG_DIR + "tn_web-ses.log",
             'formatter': 'django.ses.extra',
+            'when': 'd',
+            'utc': True,
+            'backupCount': 10,
         }
     },
     'loggers': {
         'app': {
             # 'handlers': ['console', 'mail_admins'],
-            'handlers': ['django.server'],
+            'handlers': ['django.server', 'mail_admins'],
             'level': 'INFO',
             'propagate': True,
         },
         'django': {
             # 'handlers': ['console', 'mail_admins'],
-            'handlers': ['console', 'file'],
+            'handlers': ['console', 'file', 'mail_admins'],
```

Comment:
Django's doc : 
>class AdminEmailHandler(include_html=False, email_backend=None, reporter_class=None)[[source]](https://docs.djangoproject.com/fr/4.0/_modules/django/utils/log/#AdminEmailHandler)[¶](https://docs.djangoproject.com/fr/4.0/ref/logging/#django.utils.log.AdminEmailHandler)
>
>   Ce gestionnaire envoie un courriel aux administrateurs du site ([ADMINS](https://docs.djangoproject.com/fr/4.0/ref/settings/#std-setting-ADMINS)) pour chaque message de journal qu’il reçoit.

Are you sure you want to receive an email for each log ? That seems like a lot.

Reply:
If I didn't make a mistake, it is configured only for ERROR.

Reply:
Ok, I wasn't sure because some of these loggers are INFO level.